build.gradle.kts

```diff
@@ -4,24 +4,12 @@ plugins {
 	id("io.freefair.lombok") version "8.4" apply false
 }
 
-group = "com.qpay"
-version = "0.0.1-SNAPSHOT"
```

Comment:
why did you remove group and version?

Reply:
i thought, it's optional and can be removed, but it would be good to know group of project and version, so i will return it

Reply:
I don't see that you returned it